source/base/mpi.cc

```diff
@@ -324,12 +307,12 @@ namespace Utilities
                           "in a program since it initializes the MPI system."));
 
 
-
+      int ierr;
```

Comment:
i like better the previous location. I think @drwells unified those a while ago, let's keep it homogeneous within the library.